### PR TITLE
fuck me up

### PR DESCRIPTION
**Pull request by ben-farrelly**

### CODE REVIEW

**Review by takayuki-serato — APPROVED** [review]
LGTM 👍

File: vue-lessons/requestmebruh.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+don't let me merge this
```
Comment (takayuki-serato) [inline vue-lessons/requestmebruh.txt:1]: you fucking

Comment (takayuki-serato) [inline vue-lessons/requestmebruh.txt:1]: you fucking

Comment (takayuki-serato) [reply vue-lessons/requestmebruh.txt:1]: why would you do this

Comment (takayuki-serato) [reply vue-lessons/requestmebruh.txt:1]: so disgraceful

Comment (takayuki-serato) [reply vue-lessons/requestmebruh.txt:1]: how could you call yourself an engineer?

Comment (takayuki-serato) [reply vue-lessons/requestmebruh.txt:1]: 1

Comment (takayuki-serato) [reply vue-lessons/requestmebruh.txt:1]: 2

Comment (takayuki-serato) [reply vue-lessons/requestmebruh.txt:1]: 3

Comment (takayuki-serato) [reply vue-lessons/requestmebruh.txt:1]: 4

Comment (takayuki-serato) [reply vue-lessons/requestmebruh.txt:1]: 5

Comment (takayuki-serato) [reply vue-lessons/requestmebruh.txt:1]: 6

Comment (takayuki-serato) [reply vue-lessons/requestmebruh.txt:1]: 7

Comment (takayuki-serato) [reply vue-lessons/requestmebruh.txt:1]: a

Comment (takayuki-serato) [reply vue-lessons/requestmebruh.txt:1]: a

Comment (takayuki-serato) [reply vue-lessons/requestmebruh.txt:1]: a

Comment (takayuki-serato) [reply vue-lessons/requestmebruh.txt:1]: a

Comment (takayuki-serato) [reply vue-lessons/requestmebruh.txt:1]: a

Comment (takayuki-serato) [reply vue-lessons/requestmebruh.txt:1]: a

Comment (takayuki-serato) [reply vue-lessons/requestmebruh.txt:1]: a

Comment (takayuki-serato) [reply vue-lessons/requestmebruh.txt:1]: a

Comment (takayuki-serato) [reply vue-lessons/requestmebruh.txt:1]: a

Comment (takayuki-serato) [reply vue-lessons/requestmebruh.txt:1]: a

Comment (takayuki-serato) [reply vue-lessons/requestmebruh.txt:1]: a

Comment (takayuki-serato) [reply vue-lessons/requestmebruh.txt:1]: a

Comment (takayuki-serato) [reply vue-lessons/requestmebruh.txt:1]: a

Comment (takayuki-serato) [reply vue-lessons/requestmebruh.txt:1]: a

Comment (takayuki-serato) [reply vue-lessons/requestmebruh.txt:1]: a

Comment (takayuki-serato) [reply vue-lessons/requestmebruh.txt:1]: a

Comment (takayuki-serato) [reply vue-lessons/requestmebruh.txt:1]: a

Comment (takayuki-serato) [reply vue-lessons/requestmebruh.txt:1]: a

Comment (takayuki-serato) [reply vue-lessons/requestmebruh.txt:1]: a

Comment (takayuki-serato) [reply vue-lessons/requestmebruh.txt:1]: a

Comment (takayuki-serato) [reply vue-lessons/requestmebruh.txt:1]: a

Comment (takayuki-serato) [reply vue-lessons/requestmebruh.txt:1]: a

Comment (takayuki-serato) [reply vue-lessons/requestmebruh.txt:1]: a

Comment (takayuki-serato) [reply vue-lessons/requestmebruh.txt:1]: a

Comment (takayuki-serato) [reply vue-lessons/requestmebruh.txt:1]: a

Comment (takayuki-serato) [reply vue-lessons/requestmebruh.txt:1]: a

Comment (takayuki-serato) [reply vue-lessons/requestmebruh.txt:1]: a

Comment (takayuki-serato) [reply vue-lessons/requestmebruh.txt:1]: a

Comment (takayuki-serato) [reply vue-lessons/requestmebruh.txt:1]: a

Comment (takayuki-serato) [reply vue-lessons/requestmebruh.txt:1]: a

Comment (takayuki-serato) [reply vue-lessons/requestmebruh.txt:1]: a

Comment (takayuki-serato) [reply vue-lessons/requestmebruh.txt:1]: a

Comment (takayuki-serato) [reply vue-lessons/requestmebruh.txt:1]: a

Comment (takayuki-serato) [reply vue-lessons/requestmebruh.txt:1]: a

Comment (takayuki-serato) [reply vue-lessons/requestmebruh.txt:1]: a

Comment (takayuki-serato) [reply vue-lessons/requestmebruh.txt:1]: a

Comment (takayuki-serato) [reply vue-lessons/requestmebruh.txt:1]: a

Comment (takayuki-serato) [reply vue-lessons/requestmebruh.txt:1]: a

Comment (takayuki-serato) [reply vue-lessons/requestmebruh.txt:1]: a

Comment (takayuki-serato) [reply vue-lessons/requestmebruh.txt:1]: a

Comment (takayuki-serato) [reply vue-lessons/requestmebruh.txt:1]: a

Comment (takayuki-serato) [reply vue-lessons/requestmebruh.txt:1]: a

Comment (takayuki-serato) [reply vue-lessons/requestmebruh.txt:1]: a

Comment (takayuki-serato) [reply vue-lessons/requestmebruh.txt:1]: a

Comment (takayuki-serato) [reply vue-lessons/requestmebruh.txt:1]: a

Comment (takayuki-serato) [reply vue-lessons/requestmebruh.txt:1]: a

Comment (takayuki-serato) [reply vue-lessons/requestmebruh.txt:1]: a

Comment (takayuki-serato) [reply vue-lessons/requestmebruh.txt:1]: a

Comment (takayuki-serato) [reply vue-lessons/requestmebruh.txt:1]: a

Comment (takayuki-serato) [reply vue-lessons/requestmebruh.txt:1]: a

Comment (takayuki-serato) [reply vue-lessons/requestmebruh.txt:1]: a

Comment (takayuki-serato) [reply vue-lessons/requestmebruh.txt:1]: a

Comment (takayuki-serato) [reply vue-lessons/requestmebruh.txt:1]: a

Comment (takayuki-serato) [reply vue-lessons/requestmebruh.txt:1]: a

Comment (takayuki-serato) [reply vue-lessons/requestmebruh.txt:1]: a

Comment (takayuki-serato) [reply vue-lessons/requestmebruh.txt:1]: a

Comment (takayuki-serato) [reply vue-lessons/requestmebruh.txt:1]: a

Comment (takayuki-serato) [reply vue-lessons/requestmebruh.txt:1]: a

**Review by takayuki-serato — CHANGES_REQUESTED** [review]
Not good enough

**Comment by ben-farrelly** [comment]
stop bullying me

**Comment by takayuki-serato** [comment]
G don't be a coward

**Pull request closed without merging**